janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -264,6 +265,29 @@
      */
     public static final double DEFAULT_GEO_DIST_ERROR_PCT = 0.025;
 
+    private static final String DELETION_SCRIPT = String.join(" ", "",
```

Comment:
(nitpick) This script can only be used if parameters can be used. So, shouldn't we name it `PARAMETERIZED_DELETION_SCRIPT`? (same for `ADDITION_SCRIPT` below)

Reply:
Done